test/Microsoft.ApplicationInsights.AspNetCore.Tests/Extensions/HttpRequestExtensionsTest.cs

```diff
@@ -99,7 +100,7 @@ public void TestGetUriReturnsCorrectUriIfRequestObjectSchemeAndHostAndPathAndQue
             var uri = HttpRequestExtensions.GetUri(request);
 
             Assert.Equal(
-                new Uri(string.Format("{0}://{1}{2}{3}", ExpectedSchema, ExpectedHostName, ExpectedPath, ExpectedQueryString)),
+                new Uri(string.Format(CultureInfo.InvariantCulture, "{0}://{1}{2}{3}", ExpectedSchema, ExpectedHostName, ExpectedPath, ExpectedQueryString)),
```

Comment:
No, invariant means it can't handle anything "different", it's basically English ASCII rules.